examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<p>can you add a sentence above to explain what you do in this cell brielfy? something like "let's check number of unique trips` . </p><p> </p><p>also do you need to do <code>sample(frac=1)</code> for that? can you not do <code>itineraries.utrip_id.nunique()</code> directly? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Sentence added</p>